src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+
+        if (!submitYesSamples && !submitNoSamples) {
+            FrameUtilities.showMessage(this, "No Runs", "Select either Yes or Failed runs");
+            return;
+        }
+
+        List<File> files = new ArrayList<File>();
+
+        files = QuickSubmitter.filterRuns(allFiles, submitYesSamples, submitNoSamples);
+
+        if (submitNoSamples && submitSelectedJudgingTypeRadioButton.isSelected()) {
+            if (submitSelectedJudgingTypeRadioButton.isSelected()) {
+                // if they selected the radio button for selected, they should select at least
+                if (selectedJudgingTypeIndexes == null || selectedJudgingTypeIndexes.length == 0) {
+                    FrameUtilities.showMessage(this, "No Judging Types selected", "No Judging Types selected, select at least one");
+                    return;
+                }
+
+                // fitler by judging type (accepted, wrong_answer, etc.)
+                files = ListUtilities.filterByJudgingTypes(files, submissionSolutionList);
+            }
+        }
+	    
+        if (checkBoxProblems.isSelected() ) {
+            if (submitSelectedProblemsRadioButton.isSelected()) {
+                // submissions may be filtered by problem
+
+                if (selectedProblemsIndexes == null || selectedProblemsIndexes.length == 0) {
+                    FrameUtilities.showMessage(this, "No Problem selected", "No Problems selected, select at least one");
+                    return;
+                }
+
+                // filter list by problem name
+                files = ListUtilities.filterByProblems (files, selectedProblemList);
+
+            }
+        }
+	    
+	    if (checkBoxLanguage.isSelected()) {
+	        // submissions may be filtered by language
+	        
+	        if (submitSelectedLanguagesRadioButton.isSelected()) {
+	            if (selectedLanguagesIndexes == null || selectedLanguagesIndexes.length == 0) {
+	                FrameUtilities.showMessage(this, "No Language selected", "No Languages selected, select at least one");
+	                return;
+
+	            }
+
+	            files = ListUtilities.filterByLanguages(files, getContest(), selectedLanguageList);
+	        }
+	    }
+	    
+
+	    int count = 0;
+	    for (File file : files) {
+	        count++;
+	        xlog("Will submit #" + count + " file =" + file.getAbsolutePath());
+	    }
+
+	    if (count == 0) {
+	        showMessage("There are no CDP samples source files  under " + cdpTextField.getText());
+	        return;
+	    }
+
+	    int result = FrameUtilities.yesNoCancelDialog(this, "Submit " + files.size() + " sample submissions?",
+	            "Submit CDP submissions");
+
+	    if (result == JOptionPane.YES_OPTION) {
+	        /**
+	         * This will submit each file found
+	         */
+	        submitter.sendSubmissions(files);
+
+	        showMessage("Submitted " + files.size() + " runs.");
```

Comment:
As noted elsewhere, this message is somewhere between _misleading_ and wrong.  There appear to be files in the "files" list (passed to `submitter.sendSubmissions(files)`) that do not actually get submitted.

I consider fixing this issue to be an approval showstopper.